app/code/Magento/CatalogUrlRewrite/etc/di.xml

```diff
@@ -27,6 +27,9 @@
     <type name="Magento\CatalogUrlRewrite\Model\Storage\DbStorage">
         <plugin name="dynamic_storage_plugin" type="Magento\CatalogUrlRewrite\Plugin\DynamicCategoryRewrites"/>
     </type>
+    <type name="Magento\Catalog\Model\Product\Action">
+        <plugin name="url_rewrite_product_action_plugin" type="Magento\CatalogUrlRewrite\Plugin\Catalog\Model\Product\ProductToWebsiteChange"/>
```

Comment:
Name is confusing. Please change it to something like update_url_rewrites_after_websites_update_plugin

Reply:
updated